.github/workflows/deploy.yml

```diff
@@ -0,0 +1,35 @@
+name: Deploy
+on:
+    push:
+        branches:
```

Comment:
need to change this before merging as its just set up to deploy on my fork for testing/viewing purposes.

Reply:
yep, will change it to main then.

Reply:
I removed my branch.